### PR TITLE
Sh feature/body message

### PR DESCRIPTION
**Pull request by IsmaelVillavicencio**
Originlas 
desktop 1024
mobile portrait

How it looks?
https://fiorella-dev.myshopify.com/

### CODE REVIEW

File: fiorella/src/styles/layout/_category-products-list.scss [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    z-index: 1;
+
+    @media #{ $tablet }, #{ $desktop } {
+      display: none;
+    }
+  }
+
+  .overlay {
+    background-color: $cerise-red-hover;
+    bottom: 0px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_category-products-list.scss]: `0px` should be written without units as `0`

File: fiorella/src/styles/layout/_category-products-list.scss [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    }
+
+    @media #{ $desktop } {
+      display: block;
+      font-size: 23px;
+    }
+  }
+
+  @media #{ $tablet }, #{ $desktop } {
+    margin: 0px 15px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_category-products-list.scss]: `0px` should be written without units as `0`

File: fiorella/src/styles/layout/_category-products-list.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  @media #{ $tablet }, #{ $desktop } {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 90%;
+  }
+}
+
+.category-item {
+  margin: 5px 0px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_category-products-list.scss]: `0px` should be written without units as `0`

File: fiorella/src/styles/layout/_body-message.scss [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  background: $peppermind-color;
+  padding: 60px 20px;
+  text-align: center;
+
+  .title-message {
+    overflow: hidden;
+    font-family: $playfair-display;
+    font-weight: $font-light;
+
+    &:after, &:before {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_body-message.scss]: Each selector in a comma sequence should be on its own line

File: fiorella/src/styles/layout/_body-message.scss [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    width: 150px;
+
+    @media #{ $tablet }, #{ $desktop } {
+      margin-top: 40px;
+      width: 200px;
+    }
+  }
+
+  .text-message {
+    margin: 10px 15px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_body-message.scss]: Properties should be ordered font-family, font-size, font-weight, line-height, margin

File: fiorella/src/styles/layout/_body-message.scss [hunk 6]
```diff
@@ -0,0 +1,76 @@
+.body-message {
+  background: $peppermind-color;
+  padding: 60px 20px;
+  text-align: center;
+
+  .title-message {
+    overflow: hidden;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_body-message.scss]: Properties should be ordered font-family, font-weight, overflow

File: fiorella/src/styles/layout/_body-message.scss [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    @media #{ $desktop } {
+      font-size: 20px;
+      margin: 15px 20px;
+    }
+  }
+
+  .btn-read-more {
+    font-family: $playfair-display;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_body-message.scss]: Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered display, font-family, font-weight, margin-top, padding, text-decoration

File: fiorella/src/styles/abstracts/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#E83671, 0.7);
+$cerise-red: #E83671;
+$font-color-white: #FFFFFF;
+$border-color-white: #FFFFFF;
+$peppermind-color: #F3F9EF;
```
Comment (magmahound) [inline fiorella/src/styles/abstracts/_variables.scss]: Color `#F3F9EF` should be written as `#f3f9ef`

File: fiorella/src/styles/abstracts/_variables.scss [hunk 9]
```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#E83671, 0.7);
+$cerise-red: #E83671;
+$font-color-white: #FFFFFF;
+$border-color-white: #FFFFFF;
```
Comment (magmahound) [inline fiorella/src/styles/abstracts/_variables.scss]: Color `#FFFFFF` should be written as `#ffffff`

File: fiorella/src/styles/abstracts/_variables.scss [hunk 10]
```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#E83671, 0.7);
+$cerise-red: #E83671;
+$font-color-white: #FFFFFF;
```
Comment (magmahound) [inline fiorella/src/styles/abstracts/_variables.scss]: Color `#FFFFFF` should be written as `#ffffff`

File: fiorella/src/styles/abstracts/_variables.scss [hunk 11]
```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#E83671, 0.7);
+$cerise-red: #E83671;
```
Comment (magmahound) [inline fiorella/src/styles/abstracts/_variables.scss]: Color `#E83671` should be written as `#e83671`

File: fiorella/src/styles/abstracts/_variables.scss [hunk 12]
```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#E83671, 0.7);
```
Comment (magmahound) [inline fiorella/src/styles/abstracts/_variables.scss]: Color literals like `#E83671` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E83671` should be written as `#e83671`

File: fiorella/src/styles/abstracts/_variables.scss [hunk 13]
```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#e83671, 0.7);
```
Comment (magmahound) [inline fiorella/src/styles/abstracts/_variables.scss]: Color literals like `#e83671` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: fiorella/src/sections/body-message.liquid [hunk 14]
```diff
@@ -0,0 +1,36 @@
+<div class="body-message">
+  <h1 class="title-message">{{ section.settings.titlemessage | upcase }}</h1>
+  <img class="image-message" src="{{ section.settings.imagemessage | img_url: '800x' }}" alt="{{ settings.categoryimg.alt }}" />
+  <div class="text-message">
+    <p>{{ section.settings.imagedescription }}</p>
+  </div>
+  <a href="{{section.settings.link}}" class="btn-read-more">READ MORE</a>
```
Comment (luisarmandodv) [inline fiorella/src/sections/body-message.liquid]: Can you change this text with text from locale file?

Why use locale files?
When your app supports internationalize language you need locales files, this way you app pull all text from one single file.

Example:
<p>{{ 'homepage.footer.follow_us' | t }}

**Comment by IsmaelVillavicencio** [comment]
branch problems

File: fiorella/src/sections/category-products-list.liquid [hunk 15]
```diff
@@ -0,0 +1,49 @@
+{%- if section.blocks.size > 0 -%}
```
Comment (luisarmandodv) [inline fiorella/src/sections/category-products-list.liquid]: Creo que esto no corresponde a este pull request

Comment (Ilyeo) [inline fiorella/src/sections/category-products-list.liquid]: update this file with changes from `https://github.com/magma-labs/shopify-themes/pull/75`

**Pull request closed without merging**